debian/control

```diff
@@ -0,0 +1,24 @@
+Source: libjsonv
+Priority: optional
+Maintainer: Remy Chibois <chybz@chybz.net>
+Build-Depends: debhelper (>= 9), cmake
+Standards-Version: 3.9.5
+Section: libs
+Homepage: https://github.com/tgockel/jsonv
+
+Package: libjsonv-dev
+Section: libdevel
+Architecture: any
+Depends: libjsonv1.0.0-pre0 (= ${binary:Version}), ${misc:Depends}
```

Comment:
There are two way to proceed regarding the `debian` directory.

- You include it in your source, right at the root so that any Debian user can build a package using the standard tools (which, as you might have guessed, expect this dir in the root folder)
- For non-native Debian packages, maintainers have to follow a little more detailed procedure which include the pristine source (your code untouched), a Debian specific patch archive (creating the `debian` directory and patching any files if necessary) plus a description file.

The fork I've made from your repo is in between, modifying the least amount of files to have the packaging right, immediately usable and outside the Debian "process". You can see it as a convenience modification.

Regarding the move of debian in installer it's perfectly legit and I understand you don't want to clutter your sources. I think we are both busy and I don't want to introduce mess in your work.
The easy way seems to be to move debian in installer, any Debian addict will know how to copy it in the root folder and use it from here. Unfortunately I don't have the resources (time) to properly maintain this for the Debian community and follow their procedure. I've made a package of it and uploaded to our repository so anyone can use it.

Please tell me what you like/want/forsee, this is just a starting point.